### PR TITLE
#11 UI A# login

### PR DESCRIPTION
**Pull request by skrzepto**
Basic UI login with A#

@sdimitro

### CODE REVIEW

**Comment by sdimitro** [comment]
Thank you so much Simon.

Unfortunately there are two more issues:
[1] You've introduced static/kiosk.py, which is basically a copy of
     kiosk.py from the root directory. Is there a reason behind that?
     Also, you've reintroduced requirements.txt again. When I committed
     as your last time I changed the name to DEPENDENCIES because
     it is more descriptive. requirements.txt is a pretty standard name
     but one may think that they are documentation requirements (like
     requirements from an Agile methodology and other fads)
[2] You have 6 commits for one feature, all of them not in the format
     of previous entries in the git log, which could pollute our history
     with intermediate and irrelevant commit messages.

The first issue is pretty important.
The second not so much but it can hurt in the long run.

I can fix both of them myself later if you don't feel like working on them.
It goes without saying that if that happens, I will commit under your
name and email.

If you decide to tackle them both, I would
refer to https://git-scm.com/book/en/v2
for _squashing_ (wink wink) all your commits into a single commit.

Thanks again :-)

**Comment by skrzepto** [comment]
I see that will probably be a problem in the future. Working on fixing that now

**Comment by skrzepto** [comment]
When i did an upstream fetch is messed up my local copy and kept the requiremnts.txt and

**Comment by skrzepto** [comment]
Thanks Mike for the pointers. I got the commits to squash into one. Check it out now if you could

@sdimitro

**Comment by sdimitro** [comment]
Good job man! Less clutter!

Any case you could delete the extra kiosk.py that you put by mistake in the static/ folder?

**Comment by skrzepto** [comment]
should be clean now

**Comment by sdimitro** [comment]
I put some comments on your commit.
If I'm being too anal you can tell me to stop :-P

**Comment by skrzepto** [comment]
Oh don't worry about it :) I was editing in VIM and then in PyCharm the indent spacing is different for each program. I have reformatted the code and will rename it to `sign in`

File: kiosk.py [hunk 1]
```diff
@@ -15,13 +15,27 @@
...
 app = Flask(__name__)
 
+
 @app.route('/')
 def hello():
     return render_template('hello.html')
 
+
+@app.route('/login', methods=['GET', 'POST'])
+def login():
```
Comment (sdimitro) [inline kiosk.py]: Could you change that to @app.route('/signin', ...etc) and def signin(): pretty please? :P

File: kiosk.py [hunk 2]
```diff
@@ -15,13 +15,27 @@
 # Copyright (c) 2015, Serapheim Dimitropoulos
 #
 
-from flask import Flask, render_template
+from flask import Flask, render_template, request
 
 app = Flask(__name__)
 
+
```
Comment (sdimitro) [inline kiosk.py:22]: take care of extra empty line?

**Comment by skrzepto** [comment]
The extra line is result of pep8 formating I'd say leave it as it is since my IDE formatted it for me and knows the standard better than either of us do

**Comment by sdimitro** [comment]
oh good to know, my bad then!

I will get the formatter to work with my vim setup too, thanks for pointing that out!
You are good to go!

I will merge the commit tonight or tomorrow morning the latest!
Thank you very much Simon.

**Comment by sdimitro** [comment]
Changes applied as skrzepto

**Pull request closed without merging**